pkg/applicationserver/io/pubsub/grpc.go

```diff
@@ -0,0 +1,118 @@
+// Copyright © 2019 The Things Network Foundation, The Things Industries B.V.
```

Comment:
Maybe rename this file to `grpc_webhooks.go` for consistency with package `pkg/applicationserver/io/web`?